src/lib/posts.js

```diff
@@ -157,3 +157,22 @@ export function mapPostData(post = {}) {
 
   return data;
 }
+
+/**
+ * getRelatedPosts
+ */
+
+export async function getRelatedPosts({ categories, postId }, count = 5) {
```

Comment:
since we're only ultimately getting the first instance of the category here, do you think it might make more sense to make this `category` instead of `categories`? it might make it more predictable, then when calling it, the callter would use `categories[0]`

looking below, looks like it was based off of just passing `post` in, but i think that could make it a little more clear what's happening. thoughts?

Reply:
I agree, that makes totally sense. 
When passing `post` object (and its `categories`) I was thinking that maybe in the future would be interesting to:

- Check if `categories[0]` have at least 5 posts
- In case **relatedPosts < 5 and** `categories[1]` exists ,  fill in the rest with `categories[1]`
- Update the title to have the two categories

But the way you suggest is better, because it keeps the funcionality very clear (get related posts for **one** category) and makes it very reusable.  And in case of adding extra complexity, it can be placed outside (checking relatedPosts count, calling the function for another category...). Will make the changes, thanks!!

Reply:
yeah, i think thats a cool idea though! i think we'd want to think about how that would work like title and such, maybe another Issue we can brainstorm on